cli/src/pixl_cli/main.py

```diff
@@ -66,8 +67,7 @@ def cli(*, debug: bool) -> None:
 def populate(parquet_dir: Path, *, restart: bool, queues: str) -> None:
     """
     Populate a (set of) queue(s) from a parquet file directory
```

Comment:
It's to match the metavar that click generates at the top of the help text. Confusingly (IMO) you can't document arguments (as opposed to options) in the `@click.argument` decorator; you have to do it in the main help text which it takes from your docstring. Shame it also makes a mess of the dir structure by rewrapping. I did briefly try to fix this once but didn't get very far.
```
$ pixl populate --help
Usage: pixl populate [OPTIONS] PARQUET_DIR

  Populate a (set of) queue(s) from a parquet file directory PARQUET_DIR:
  Directory containing the public and private parquet input files and an
  extract_summary.json log file.     It's expected that the directory
  structure will be:

          PARQUET-DIR         ├── private         │   ├── PERSON_LINKS.parquet
          │   └── PROCEDURE_OCCURRENCE_LINKS.parquet         ├── public
          │   └── PROCEDURE_OCCURRENCE.parquet         └──
          extract_summary.json

Options:
  --queues TEXT             Comma seperated list of queues to populate with
                            messages generated from the input file(s)
                            [default: pacs,ehr]
  --restart / --no-restart  Restart from a saved state. Otherwise will use the
                            given input file(s)  [default: restart]
  --help                    Show this message and exit.

```

Reply:
Re your first comment, the other args are documented in the `click.option` decorators. I don't think you can do it the normal way in the docstring without it showing up in the help text in a weird way.

Reply:
aha, yes okay, ignore me then!